### DISCUSSION TITLE
Problems that occur during the use of ctcreate！why？please

### DISCUSSION THREAD

**Discussion by shirley32123** [nrc-cnrc/EGSnrc#1110] [Questions]
![Uploading 微信图片_20240404160522.png…]()

why？
I've tried every possible error, but I've failed.

**Comment by shirley32123** [comment]
![微信图片_20240404160522](https://github.com/nrc-cnrc/EGSnrc/assets/165998867/f6de9623-6679-4e7f-be94-dacd960e4fb0)

**Reply by rtownson** [reply]
Can you share your input file for ctcreate? Note that the DICOM list file that you provide by name in the input file should be a text file, that contains a list of DICOM files, one per line.

**Reply by shirley32123** [reply]
![1](https://github.com/nrc-cnrc/EGSnrc/assets/165998867/472cdd34-078c-4aa1-a54c-458bab7d5844)
![2](https://github.com/nrc-cnrc/EGSnrc/assets/165998867/cabbd187-9b54-4e35-821a-030257958a86)
![3](https://github.com/nrc-cnrc/EGSnrc/assets/165998867/9f974d96-8e9f-4a4b-ab64-be81abeed7c2)
I've put together three pictures. Is the third one what you want?I changed it a little: I deleted the spaces in the folder, but it still didn't work.

**Reply by rtownson** [reply]
Please also share your .inp file. In your .ctlst file (your last image), make sure to add a blank line at the end of the file. It might also be helpful if you could copy/paste all of the output from running ctcreate, rather than just a screenshot of the last few lines.

**Reply by shirley32123** [reply]
Here's the full information for the .ctlst file: I don't understand what you mean by adding a blank line at the end of the file?

```
=============================================================
 Running ctcreate
=============================================================


 ************************************************************
 
           CT formats currently supported:
     ------------------------------ 
 
           1. Pinnacle                    
           2. CADPLAN 
           3. DICOM                       


 ************************************************************


 Input the format of your CT data
 :           /home/LUNG1-001/RTdose Unity/RTdose Unity/Bm12.dcm

This is a dividing line. Here's all the output from ctcreate:

eant4@geant4:~/LUNG1-001/RTdoseUnity/RTdoseUnity$ ctcreate

No input file -> all input to come from the terminal

=============================================================
 Running ctcreate
=============================================================


 ************************************************************
 
           CT formats currently supported:
     ------------------------------ 
 
           1. Pinnacle                    
           2. CADPLAN 
           3. DICOM                       


 ************************************************************


 Input the format of your CT data
 : 3
                                                           3


 Input the full name of the file of DICOM file names
 : 101
101

 DICOM data is little endian.
 Machine is little endian...bytes will not be swapped

 Image slices being ordered in increasing Z
.rror opening file /home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm12.dcm

Program received signal SIGSEGV: Segmentation fault - invalid memory reference.

Backtrace for this error:
#0  0x7f6b6e8a7d4a
#1  0x7f6b6e8a6ee5
#2  0x7f6b6e5890bf
#3  0x7f6b6e5c8eb5
#4  0x55f4b59cfba7
#5  0x55f4b59d056a
#6  0x55f4b59ce2b6
#7  0x55f4b59c035e
#8  0x7f6b6e56a0b2
#9  0x55f4b59c039d
#10  0xffffffffffffffff
段错误 (核心已转储)
geant4@geant4:~/LUNG1-001/RTdoseUnity/RTdoseUnity$ 
```

Also, I've uploaded the .inp file in the attachment.

**Reply by shirley32123** [reply]
Sorry, I can't upload the .inp file. The platform does not support such files.“Try again with GIF, JPEG, JPG, MOV, MP4, PNG, SVG, WEBM, CPUPROFILE, CSV, DMP, DOCX, FODG, FODP, FODS, FODT, GZ, JSON, JSONC, LOG, MD, ODF, ODG, ODP, ODS, ODT, PATCH, PDF, PPTX, TGZ, TXT, XLS, XLSX or ZIP.”

/home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm12.dcm
/home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm11.dcm
/home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm10.dcm
/home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm9.dcm
/home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm8.dcm
/home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm7.dcm
/home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm6.dcm
/home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm5.dcm
/home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm4.dcm
/home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm3.dcm
/home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm2.dcm
/home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm1.dcm

**Reply by ftessier** [reply]
You can change the extension of the file from `.inp` to `.txt` to enable uploading here.

**Reply by rtownson** [reply]
Does `~/` really just point to `/home/`? The error you are seeing makes it sound like the file `/home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm12.dcm` does not exist or doesn't have read permissions. What output do you get for `ls -lh /home/LUNG1-001/RTdoseUnity/RTdoseUnity/Bm12.dcm`?
